haiku-apps/bic/bic-1.0.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="WYSIWYG GUI designer"
+DESCRIPTION="BeInterfaceCreator is WYSIWYG GUI designer for Haiku."
+HOMEPAGE="https://github.com/HaikuArchives/BeInterfaceCreator"
+COPYRIGHT="Unknown"
+LICENSE="GPL"
+REVISION="1"
+srcGitRev=""
```

Comment:
Well you need to fill srcGitRev for the recipe to make sense.

Reply:
I thought that I would fill it later. Oh well